src/main/java/rs/teslaris/core/controller/ProceedingsPublicationController.java

```diff
@@ -24,13 +29,25 @@ public class ProceedingsPublicationController {
 
     private final ProceedingsPublicationService proceedingsPublicationService;
 
+    private final JwtUtil tokenUtil;
+
+    private final UserService userService;
+
 
     @GetMapping("/{publicationId}")
     public ProceedingsPublicationDTO readProceedingsPublication(
         @PathVariable Integer publicationId) {
         return proceedingsPublicationService.readProceedingsPublicationById(publicationId);
     }
 
+    @GetMapping("/for-event/{eventId}")
+    public List<ProceedingsPublicationResponseDTO> readProceedingsPublicationsForEvent(
+        @PathVariable Integer eventId, @RequestHeader("Authorization") String bearerToken) {
+        return proceedingsPublicationService.findProceedingsForEvent(eventId,
```

Comment:
This should return conference articles (ProceedingsPublication) from a certain event (conference) which author is loggedIn researcher, right? If that is the case, not sure the name of method is accurate. Does this return proceedings or proceedings publications?

Reply:
This returns ProceedingsPublications.

Reply:
I renamed it to `readMyProceedingsPublicationsForEvent()` Is that ok?